packages/block-editor/src/components/block-list-appender/index.js

```diff
@@ -30,19 +31,32 @@ function BlockListAppender( {
 	rootClientId,
 	canInsertDefaultBlock,
 	isLocked,
-	renderAppender: CustomAppender,
+	renderAppender: CustomAppender, // Deprecated.
+	appender,
 	className,
 	selectedBlockClientId,
 	tagName: TagName = 'div',
 } ) {
-	if ( isLocked || CustomAppender === false ) {
+	if ( undefined !== CustomAppender ) {
```

Comment:
Not that it makes a big difference, but the preference in Gutenberg is not to use yoda conditions, because the linting here catches any issues.

Reply:
Ah, perfect! Updated here: https://github.com/WordPress/gutenberg/pull/29911/commits/b2d0defbe249855354ad29b598bbd4d99553ef88.
TIL 😀